epub34/a11y-tech/index.html

```diff
@@ -1128,8 +1113,8 @@ <h4>Provide page break markers</h4>
 						list</a> is the only way a user can jump to the locations.</p>
 
 					<aside class="example" title="Adding a hyperlink destination in EPUB 2">
-						<p>In this example, the EPUB creator adds an XHTML 1.1 <code>span</code> element to use as a
-							hyperlink destination.</p>
+						<p>In this example, an XHTML 1.1 <code>span</code> element is used as a hyperlink
+							destination.</p>
```

Comment:
```suggestion
						<p>In this example, an XHTML <code>span</code> element is used as a hyperlink
							destination.</p>
```

With an eye on the discussion about non-XML syntax to HTML, is it wise to refer to XHTML 1.1 in this document? Isn't better to use the XHTML term only? (That may be true elsewhere in the document, not affected by this PR.)

Reply:
This is explicitly an example of EPUB 2 markup which is why xhtml 1.1 is called out. It's not how you would add a page break to an epub 3 file, as you have both the epub:type and role attributes at your disposal.

Reply:
Ah o.k., that is fine then.